apps/dashboard/app/controllers/files_controller.rb

```diff
@@ -6,11 +6,13 @@ def fs
     request.format = 'json' if request.headers['HTTP_ACCEPT'].split(',').include?('application/json')
 
     @path = normalized_path
+
     AllowlistPolicy.default.validate!(@path)
 
+    path = PosixFile.new(@path)
```

Comment:
I don't know if I like having both the member variable `@path` and the method scoped variable `path`. As you can see the member variable `@path` is being used in the views and more in this method.

Can we make it so `@path = PosixFile.new(normalized_path)` and `delegate` however many of those methods to _it's_ member variable (like `@path.descend` used in the view).